src/Components/OATPropertyEditor/Internal/PropertyList/Internal/PropertyListItem/Internal/PropertyListItemChildHost/Internal/PropertyListItemMapChild/PropertyListItemMapChild.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+    IPropertyListItemMapChildProps,
+    IPropertyListItemMapChildStyleProps,
+    IPropertyListItemMapChildStyles
+} from './PropertyListItemMapChild.types';
+import { getStyles } from './PropertyListItemMapChild.styles';
+import { classNamesFunction, styled } from '@fluentui/react';
+import { useExtendedTheme } from '../../../../../../../../../../Models/Hooks/useExtendedTheme';
+import PropertyListItem from '../../../../PropertyListItem';
+
+const getClassNames = classNamesFunction<
+    IPropertyListItemMapChildStyleProps,
+    IPropertyListItemMapChildStyles
+>();
+
+const PropertyListItemMapChild: React.FC<IPropertyListItemMapChildProps> = (
+    props
+) => {
+    const { item, indexKey, level, styles } = props;
+
+    // contexts
+
+    // state
+
+    // hooks
+
+    // callbacks
+
+    // side effects
+
+    // styles
+    const classNames = getClassNames(styles, {
+        theme: useExtendedTheme()
+    });
+
+    return (
+        <div className={classNames.root}>
+            <PropertyListItem
+                indexKey={`${indexKey}.0`}
```

Comment:
ummmm. I think that was to make the child have a different key than the parent so that we can build a sort of path to the control that can be accessed in tests. I don't think it's used for functionality though.

Reply:
But if it is not dynamic, won't that affect child items that are more than one level down?